pkg/kube/client.go

```diff
@@ -401,7 +401,19 @@ func NewClient(clientConfig clientcmd.ClientConfig) (Client, error) {
 	return newClientInternal(newClientFactory(clientConfig), "")
 }
 
+// NewDefaultClient creates a Kubernetes client from the current Kubernetes context.
```

Comment:
Do we need this ? Seems strange to not have needed it so far.

Reply:
no, was for debugging